src/dsp_tools/connection/connection_live.py

```diff
@@ -7,6 +7,9 @@
 import requests
 
 from dsp_tools.models.exceptions import BaseError
+from dsp_tools.utils.create_logger import get_logger
+
+logger = get_logger(__name__)
```

Comment:
I realise now, I ended up removing all uses of this logger again... But probably the connection should log whenever it's doing something, right?

Reply:
This approach comes with the usual issue, that when something went wrong that you did not forsee, you don't have the diagnostics, unless you run the process again (which may take long, or - even worse - can't be done because you were uploading to Prod and might risk duplicating something).  
But that's a separate issue we can discuss some other time.